packages/app/src/shared/hooks/use-get-balance/decoders.ts

```diff
@@ -0,0 +1,20 @@
+import * as SorobanClient from 'soroban-client'
+
+const valueToI128String = (value: SorobanClient.xdr.ScVal) =>
+  SorobanClient.scValToBigInt(value).toString()
+
+export const decodei128 = (b64: string) => {
+  const value = SorobanClient.xdr.ScVal.fromXDR(b64, 'base64')
+  try {
+    return valueToI128String(value)
+  } catch (error) {
```

Comment:
We can don't add arguments that we don't use.
```suggestion
  } catch {
```